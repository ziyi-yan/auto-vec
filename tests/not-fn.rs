use auto_vec::auto_vec;

#[auto_vec]
pub struct foo {
    x: i32,
    y: String,
}
fn main() {}
