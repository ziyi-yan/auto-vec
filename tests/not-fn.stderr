error: expected `fn`
 --> $DIR/not-fn.rs:4:5
  |
4 | pub struct foo {
  |     ^^^^^^
